lib/widgets/custom_drawer.dart

```diff
@@ -112,37 +113,17 @@ class CustomDrawer extends StatelessWidget {
                             .strictTranslate("Join new Organization")),
                       ),
                       ListTile(
-                        onTap: () async {
-                          await showDialog(
-                            context: context,
-                            builder: (context) {
-                              return AlertDialog(
-                                actions: [
-                                  TextButton(
-                                    onPressed: () => Navigator.pop(context),
-                                    child: const Text("Close"),
-                                  ),
-                                  TextButton(
-                                    onPressed: () {
-                                      //Exit organization
-                                    },
-                                    child: const Text(
-                                      "Exit",
-                                      style: TextStyle(color: Colors.red),
-                                    ),
-                                  ),
-                                ],
-                                title: const Text(
-                                  "Confirmation",
-                                  style: TextStyle(
-                                      color: Colors.white, fontSize: 22),
-                                ),
-                                content: const Text(
-                                    "Are you sure you want to exit this organization?"),
-                              );
+                        onTap: () => navigationService.pushDialog(
+                          CustomAlertDialog(
+                            reverse: true,
+                            dialogSubTitle:
+                                'Are you sure you want to exit this organization?',
+                            successText: 'Exit',
+                            success: () {
+                              //Exit org
```

Comment:
It seems like the leave functionality is to be implemented.

Reply:
Yes. The work is yet going on at the backend. I mentioned it in the issues but forgot to mention here in the conversation.

Reply:
https://github.com/PalisadoesFoundation/talawa/issues/984#issuecomment-916703158